### DISCUSSION TITLE
English UK language ?

### DISCUSSION THREAD

**Discussion by HippoBloke** [jonaswinkler/paperless-ng#556] [Archived FRs]
Any chance of English UK - the US dates drive me insane :)

Thanks for the superb work

Hippo

**Comment by jonaswinkler** [comment]
Absolutely; It's just that I've got some issues with paperless still using en-US even when en-GB is available and selected in the settings.. Need to figure that out.

**Reply by HippoBloke** [reply]
Thanks, I’ll keep an eye out 👍🏻

**Reply by HippoBloke** [reply]
Hi Jonas

Saw the update today and can confirm GB local works and I can now un-knot my brain. 

Many thanks.

**Reply by jonaswinkler** [reply]
Sure thing. I'm aware that the date inputs and their format are still an issue (their actual behavior depends on the browser).
